x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/customization_modal.test.tsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { setMockValues, setMockActions } from '../../../../__mocks__';
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiButton, EuiButtonEmpty } from '@elastic/eui';
```

Comment:
optional newline nit, group 3rd party imports

```suggestion
import React from 'react';
import { shallow } from 'enzyme';
import { EuiButton, EuiButtonEmpty } from '@elastic/eui';
```

Reply:
40a9de5a10838bb64754861f5d825390aacbbc8b